README.md

```diff
@@ -2,40 +2,74 @@
 
 ![](https://img.shields.io/badge/Kubernetes-1.15-326de6.svg) ![Build Status](https://travis-ci.org/ubuntu/microk8s.svg?branch=master)
 
-<img src="/docs/images/certified_kubernetes_color-222x300.png" align="right" width="200px">Kubernetes in a [snap](https://snapcraft.io/) that you can run locally.
+<img src="/docs/images/certified_kubernetes_color-222x300.png" align="right" width="200px">Kubernetes in a [snap](https://snapcraft.io/).
 
-You can install MicroK8s with the latest stable upstream Kubernetes release with:
+Dead simple to install, fully featured, always fresh, upstream Kubernetes available in 42 Linux distributions. Prefect for:
 
-```
-snap install microk8s --classic
-```
+- your workstation,
+- CI/CD, and
+- the edge
+
+To quote [Kelsey Hightower](https://twitter.com/kelseyhightower/status/1120834594138406912), `... Canonical might have assembled the easiest way to provision a single node Kubernetes cluster`.
+
+## How does it work?
+
+- Being a [snap](https://snapcraft.io/microk8s), MicroK8s targets 42 Linux distributions.
+
+- MicroK8s releases happen the same day as upstream K8s.
 
-For more information on using MicroK8s see the [official docs](https://microk8s.io/docs/).
+- Snap updates are seamlessly delivered keeping your cluster up-to-date.
 
+- Dependencies are included in the 200MB snap package.
 
+- All K8s versions from v1.10 onwards as well as alpha, beta and release candidates are available for testing your workload with.
+
+- A curated collection of manifests is maintained for your convenience. You can enable DNS, dashboard, ingress, istio, knative in a single command.
+
+
+## Quickstart
+
+Deploy MicroK8s with:
+
+```
+snap install microk8s --classic
+```
 
-## Building from source
+To avoid colliding with a `kubectl` already installed and to avoid overwriting any existing Kubernetes configuration file, MicroK8s adds a `microk8s.kubectl` command, configured to exclusively access the MicroK8s cluster. When following instructions online, make sure to prefix `kubectl` with `microk8s.`.
 
-To build the snap you need a [working LXD](https://linuxcontainers.org/lxd/getting-started-cli/#snap-package-archlinux-debian-fedora-opensuse-and-ubuntu) installation. To install LXD on Ubuntu first remove any old packages:
 ```
-sudo apt-get purge lxc*
-sudo apt-get purge lxd*
+microk8s.kubectl get nodes
+microk8s.kubectl get services
 ```
 
-Get the latest LXD and configure it with:
+If you already have `kubectl` installed and you want to use it to access the MicroK8s deployment you can export the cluster's config with:
```

Comment:
Would prefer:

To use MicroK8s as your default cluster with your installed `kubectl`, do this: